Firmware/printer.cfg

```diff
@@ -0,0 +1,275 @@
+# mini with that SD card.
+
+# See the example.cfg file for a description of available parameters.
+
+
+
+# Required include for Mainsail
+[include mainsail.cfg]
+
+
+[mcu]
+##	Obtain mcu value by "ls -l /dev/serial/by-id/" 
+serial: /dev/serial/by-id/usb-Klipper_stm32f103xe_33FFDB055356343509851943-if00
+
+
+[stepper_x]
+step_pin: PB13
+dir_pin: PB12           # Add ! if moving opposite direction
+enable_pin: !PB14
+rotation_distance: 40
+microsteps: 16
+endstop_pin: PC0 
+position_endstop: 120
+position_max: 120
+homing_speed: 20        # Increase after initial setup, Max 100
+homing_retract_dist: 5
+homing_positive_dir: true
+
+[tmc2209 stepper_x]
+uart_pin: PC11
+tx_pin: PC10
+uart_address: 0
+interpolate: True
+run_current: .5         # For V0.1 spec NEMA14 40Ncm
+hold_current: .25
+sense_resistor: 0.110
+stealthchop_threshold: 500
+
+
+[stepper_y]
+step_pin: PB10
+dir_pin: PB2            # Add ! if moving opposite direction
+enable_pin: !PB11
+rotation_distance: 40
+microsteps: 16
+endstop_pin: PC1
+position_endstop: 120
+position_max: 120
+homing_speed: 20        # Increase after initial setup, Max 100
+homing_retract_dist: 5
+homing_positive_dir: true
+
+[tmc2209 stepper_y]
+uart_pin: PC11
+tx_pin: PC10
+uart_address: 2
+interpolate: True
+run_current: .5         # For V0.1 spec NEMA14 40Ncm
+hold_current: .25
+sense_resistor: 0.110
+stealthchop_threshold: 500
+
+
+[stepper_z]
+step_pin: PB0
+dir_pin: !PC5           # Remove ! if moving opposite direction
+enable_pin: !PB1
+rotation_distance: 8    # For T8x8 integrated lead screw
+microsteps: 16
+endstop_pin: PC2
+position_endstop: -.10
+position_max: 120
+position_min: -2
+homing_speed: 10    # Max 100
+second_homing_speed: 3.0
+homing_retract_dist: 3.0
+
+[tmc2209 stepper_z]
+uart_pin: PC11
+tx_pin: PC10
+uart_address: 1
+interpolate: True
+run_current: 0.37       # For V0.1 NEMA17 w/ integrated lead screw
+hold_current: 0.35
+sense_resistor: 0.110
+stealthchop_threshold: 500
+
+
+[extruder]
+step_pin: PB3
+dir_pin: PB4            # Add ! if moving opposite direction
+enable_pin: !PD2
+full_steps_per_rotation: 200 # 1.8deg motor
+rotation_distance: 22.67895  # For 5mm shaft driven Bondtech gearsets
+gear_ratio: 50:17       # For v0.1 Afterburner Mini
+microsteps: 16
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PC8
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PA0
+control: pid            # Tune with `M106 S64` then `PID_CALIBRATE HEATER=extruder TARGET=245`
+pid_Kp: 28.182
+pid_Ki: 1.978
+pid_Kd: 100.397
+min_temp: 0
+max_temp: 270
+min_extrude_temp: 170
+max_extrude_only_distance: 780.0
+max_extrude_cross_section: .8
+pressure_advance: 0.0
+pressure_advance_smooth_time: 0.040
+
+[tmc2209 extruder]
+uart_pin: PC11
+tx_pin: PC10
+uart_address: 3
+interpolate: True
+run_current: 0.3
+hold_current: 0.2
+sense_resistor: 0.110
+stealthchop_threshold: 500
+
+
+[heater_bed]
+heater_pin: PC9
+sensor_type: NTC 100K beta 3950
```

Comment:
Perhaps another comment like `# This is the sensor Keenovo uses, verify for your heater.`